code/lightna/magento-os-frontend/layout/page.yaml

```diff
@@ -32,8 +32,15 @@ template: page.phtml
       attributes:
         class:
           - header__container
-      .mobile-menu-toggle:
-        template: page/header/mobile-menu-toggle.phtml
+      .header-left:
```

Comment:
it's already working a bit :+1: 
will suggestions also work? :)

Reply:
a bit? 😃 
can I show them by my own?

I am planning to add search history as next step / thinking about 'trending now' queries

Reply:
trending - I guess, can be info from Top Search Terms
history - no, I just liked in other shop and wanted to add, would you like me not too?

Reply:
yep, found it - `/search/ajax/suggest`
separate PR maybe?

Reply:
up to you

Reply:
than let's merge search first and add suggestions after